Automattic/jetpack
# Business hours: Add unit tests and fixtures for the business hours block

Adds test fixtures for the Business Hours block and unit tests for the block's Edit component. Also updates the Remove Hours button to include label/tooltip text.




#### Changes proposed in this Pull Request:

* Add block fixture parsing tests for the Business hours block (using the default hours, and using custom hours)
* Add unit tests for the block's Edit component
* Add label text to the Remove Hours button, so that it's more accessible (tooltip text makes it clearer that you're removing the hours, not the entire day) and cleaner to test (look for the button by its label text instead of by its class name)

![image](https://user-images.githubusercontent.com/14988353/110583661-20de2500-81c2-11eb-982d-b32b6f9a9807.png)

#### Jetpack product discussion


See p1HpG7-b3G-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Change directory to `cd projects/plugins/jetpack` and run the following test making sure they all pass:
* `yarn jest extensions/blocks/business-hours/test/`
* In a browser, insert the Business Hours block, and add an additional set of hours to one of the days. Hover over the trashcan icon to remove hours and you should see tooltip text saying "Remove Hours".

#### Proposed changelog entry for your changes:


* Business Hours Block: Add tooltip text to the Remove Hours button.
